### PR TITLE
Jobs are not refreshed when changing a job from the pipeline page(#2a7yg9j)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues
 

 Closes https://github.com/hotwax/threshold-management/issues/137

 ### Short Description and Why It's Useful
 

 ### Screenshots of Visual Changes before/after (If There Are Any)
 

 **IMPORTANT NOTICE** - Remember to add changelog entry

 ### Contribution and Currently Important Rules Acceptance
 

 - [ ] I read and followed [contribution rules](https://github.com/hotwax/threshold-management#contribution-guideline)

### CODE REVIEW

File: src/store/modules/job/actions.ts [hunk 1]
```diff
@@ -446,6 +448,8 @@ const actions: ActionTree<JobState, RootState> = {
             'systemJobEnumId_op': 'equals'
           }
         })
+        const jobEnums = process.env?.VUE_APP_JOB_ENUMS ? JSON.parse(process.env?.VUE_APP_JOB_ENUMS) : [];
```
Comment (ymaheshwari1) [inline src/store/modules/job/actions.ts:451]:
```suggestion
        const jobEnums = process.env?.VUE_APP_JOB_ENUMS ? JSON.parse(process.env.VUE_APP_JOB_ENUMS) : [];
```

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/components/JobConfiguration.vue [hunk 2]
```diff
@@ -114,7 +114,7 @@ import {
   personCircleOutline
 } from "ionicons/icons";
 import { mapGetters, useStore } from "vuex";
-import { handleDateTimeInput } from "@/utils";
+import { handleDateTimeInput, hasError } from "@/utils";
```
Comment (adityasharma7) [inline src/components/JobConfiguration.vue:117]: I think there are unwanted changes in this file. Please revert

File: src/views/ThresholdUpdates.vue [hunk 3]
```diff
@@ -253,7 +253,7 @@
         </section>
 
         <aside class="desktop-only" v-show="segmentSelected === 'pending' && currentJob">
-          <JobConfiguration :title="title" :job="currentJob" :productCount="productCount" :status="currentJobStatus" :type="freqType" :key="currentJob"/>
+          <JobConfiguration :title="title" :job="currentJob" :productCount="productCount" :status="currentJobStatus" :type="freqType" :key="currentJob" :jobEnums="jobEnums" />
```
Comment (adityasharma7) [inline src/views/ThresholdUpdates.vue:256]: I think this may not be needed as we are not using jobEnums in JobConfiguration component

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

**Comment by ymaheshwari1** [comment]
closes #137

File: src/components/JobConfiguration.vue [hunk 4]
```diff
@@ -204,9 +206,25 @@ export default defineComponent({
           }, {
             text: this.$t('Skip'),
             handler: () => {
-              if (job) {
-                this.store.dispatch('job/skipJob', job)
+              let skipTime = {};
```
Comment (adityasharma7) [inline src/components/JobConfiguration.vue:209]: I think this logic should belong to action only
Also, we followed same in Job Manager so would be good for consistency
https://github.com/hotwax/job-manager/blob/main/src/store/modules/job/actions.ts#L562

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

**Comment by ymaheshwari1** [comment]
Fixed in #167
Closing

**Pull request closed without merging**